src/platform/packages/shared/response-ops/alerts-table/components/empty_state.tsx

```diff
@@ -45,7 +61,65 @@ export const EmptyState: React.FC<
   additionalToolbarControls,
   alertsQuerySnapshot,
   showInspectButton,
+  error,
+  onResetToPreviousState,
 }) => {
+  const renderErrorState = () => (
+    <EuiFlexItem>
+      <EuiText color="danger" css={textStyle}>
+        {error?.message}
+      </EuiText>
+      <EuiSpacer size="m" />
+
+      {onResetToPreviousState ? (
+        <EuiFlexGroup justifyContent="flexStart">
+          <EuiFlexItem grow={false}>
+            <EuiButton
+              onClick={onResetToPreviousState}
+              size="m"
+              data-test-subj="resetToPreviousStateButton"
+            >
+              <FormattedMessage
+                id="xpack.triggersActionsUI.empty.resetButton"
+                defaultMessage={'Reset sort'}
```

Comment:
This works for now because we only have this logic for sorting errors, but in a scenario where we have better handling for this table, the existence of `onResetToPreviousState` does not immediately imply a sorting error.

Reply:
How about `onResetSortToPreviousState`?